Dockerfile.Windows

```diff
@@ -1,4 +1,7 @@
-FROM mcr.microsoft.com/windows/servercore:1809 as core
+ARG BASE_IMAGE
+ARG BASE_IMAGE_TAG
+
+FROM mcr.microsoft.com/windows/${BASE_IMAGE}:${BASE_IMAGE_TAG} as core
```

Comment:
Does registry also need to be configurable?

Reply:
for now, not really. But I added it in case and set default value